rpc/README.md

```diff
@@ -1 +1,10 @@
-# RPC Package
\ No newline at end of file
+# RPC
+
+Package contains other subpackages with structures that can be utilized in your `proto` definitions.
+Bolow is the breaf desxription for each subpackage. More details are available inside correspondent subpackage.
+
+- [errdetails](errdetails) - package contains `proto` representation for `deltails` section inside [error response](../errors).
+
+- [errfields](errfields) - package contains `proto` representation for `fields` section inside [error response](../errors).
```

Comment:
It might be nice to wrap these package names in backticks (see the top-level README [here](https://github.com/infobloxopen/atlas-app-toolkit/blob/refactor-readme/README.md#request-handling-tools)).